pom.xml

```diff
@@ -129,7 +144,7 @@
 					<assemblyArtifact>
 						<groupId>com.ibm.websphere.appserver.runtime</groupId>
 						<artifactId>wlp-kernel</artifactId>
-						<version>17.0.0.1</version>
+						<version>17.0.0.2</version>
```

Comment:
are you getting a warning message because of no version for the plugin itself?  line 142?